### PR TITLE
fix some format errors

### PR DESCRIPTION
**Pull request by zhenghenghuo**
### What I have done?
fix some format errors
Signed-off-by: zhenghenghuo <zhenghenghuo@zju.edu.cn>

### CODE REVIEW

File: contributors/design-proposals/ha_master.md [hunk 1]
```diff
@@ -183,10 +183,10 @@ A pseudo-code for adding a new master replica using managed DNS and a loadbalanc
   4. Update DNS to point to the load balancer.
 2. Clone existing master (create a new VM with the same configuration) including
    all env variables (certificates, IP ranges etc), with the exception of
-   `INITIAL_ETCD_CLUSTER`.
+   INITIAL_ETCD_CLUSTER.
```
Comment (pigmej) [inline contributors/design-proposals/ha_master.md]: \` were needed there, it makes gfm to render them correctly like: `INITIAL_ETCD_CLUSTER`

File: contributors/design-proposals/ha_master.md [hunk 2]
```diff
@@ -183,10 +183,10 @@ A pseudo-code for adding a new master replica using managed DNS and a loadbalanc
...
 3. SSH to an existing master and run the following command to extend etcd cluster
    with the new instance:
-   `curl <existing_master>:4001/v2/members -XPOST -H "Content-Type: application/json" -d '{"peerURLs":["http://<new_master>:2380"]}'`
+   curl <existing_master>:4001/v2/members -XPOST -H "Content-Type: application/json" -d '{"peerURLs":["http://<new_master>:2380"]}'
```
Comment (pigmej) [inline contributors/design-proposals/ha_master.md]: ` were needed there, it makes gfm to render them correctly like:  `curl <existing_master>:4001/v2/members -XPOST -H "Content-Type: application/json" -d '{"peerURLs":["http://<new_master>:2380"]}'`

File: contributors/design-proposals/ha_master.md [hunk 3]
```diff
@@ -215,7 +215,7 @@ A pseudo-code for deleting an existing replica for the master is the following:
 ```
 1. Remove replica IP address from the load balancer or DNS configuration
 2. SSH to one of the remaining masters and run the following command to remove replica from the cluster:
-  `curl etcd-0:4001/v2/members/<id> -XDELETE -L`
+  curl etcd-0:4001/v2/members/<id> -XDELETE -L
```
Comment (pigmej) [inline contributors/design-proposals/ha_master.md]: and same there

File: contributors/design-proposals/ha_master.md (hunk 1 above)
Comment (pigmej) [inline contributors/design-proposals/ha_master.md]: Yeah but then everything looks the same and it's hard to see what's code in that pseudo-code and what is not. But this is my opinion ;)

**Comment by philips** [comment]
Closing because the backticks are intentional.

**Pull request closed without merging**